src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/IndividualRepo.java

```diff
@@ -459,6 +487,33 @@ public SolrQuery query() {
             return this.query;
         }
 
+        public JsonQueryRequest jsonQuery(List<String> ids) {
+            final ModifiableSolrParams params = new ModifiableSolrParams();
+            params.set("q.op", defaultOperator);
+
+            JsonQueryRequest request = new JsonQueryRequest(params)
+                .setQuery(DEFAULT_QUERY)
+                .setLimit(this.query.getRows());
+
+            String termFilter = String.format("{!terms f=id}:%s", String.join(",", ids)); 
+
+            request.withFilter(termFilter);
+
+            // there is variation of boolean logic for filtering
+            // for export filters grouped by field are OR'd together
+            for (String filter : this.filters) {
+                request.withFilter(filter.replace(" AND ", " OR "));
```

Comment:
This is modifying an assumption hardcoded above. Whether or not to group like fields together using OR or AND. AND work well on repeatable fields. Such as A repeatable field containing similar sets as another document. This is an area that could use improvement.